app/src/main/java/ch/epfl/cs311/wanderwave/model/auth/AuthenticationController.kt

```diff
@@ -140,6 +140,10 @@ constructor(
     auth.signOut()
   }
 
+  suspend fun getAccessToken(): String? {
+    return tokenRepository.getAuthToken(AuthTokenRepository.AuthTokenType.SPOTIFY_ACCESS_TOKEN)
+  }
+
```

Comment:
You can also remove this function

Reply:
I will keep it only for testing , I removed it from the function but I am using it for tests